packages/@magic-sdk/types/src/modules/auth-types.ts

```diff
@@ -43,6 +43,29 @@ export interface LoginWithSmsConfiguration {
   phoneNumber: string;
 
   /**
+   * When `true`, a pre-built modal interface will show to the user, directing
+   * them to check their SMS for the one time passcode (OTP) to complete their
+   * authentication.
+   *
+   * When `false`, developers will be able to implement their own custom UI to
+   * continue the SMS OTP flow.
+   */
+  showUI?: boolean;
+
+  /**
+   * Device Unrecognized UI will enforce showing up to secure user's login
+   *
+   * When set to true (default), an improved device recognition UI will be displayed to the user,
+   * prompting them to verify their login by checking their email for device approval. This feature
+   * enhances authentication security.
+   *
+   * This param will only be affect if showUI is false. When set to false,
+   * developers have the flexibility to implement their own customized UI to
+   * handle device check events, providing a more tailored user experience.
+   */
+  deviceCheckUI?: boolean;
```

Comment:
We might not need this, deviceCheckUI is a temporary flag for migration purpose when the original flow has whitelabel built in. 

Since SMS doesn't have whitelabel flow in the first place, we can simply hide the entire UI by just using showUI